### PR TITLE
SpringDAO

### PR DESCRIPTION
**Pull request by AlexN-OceanDAO**
**Add space for SpringDAO**  
  
Etherscan: https://etherscan.io/address/0x6D40A673446B2D00D1f9E85251209C638049ba22  
Website: https://oceanprotocol.com/ (Ocean Protocol v.Alpha DAO)
Twitter: https://twitter.com/TheSpringDAO

### CODE REVIEW

File: skins/spring-dao.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.example {
+  --primary-color: #384aff;
```
Comment (bonustrack) [inline skins/spring-dao.scss:2]: Are you using the default theme? If yes you don't need this file and you can remove "skin" param in the index.json file

**Review by AlexN-OceanDAO — COMMENTED** [review]
Changes updates

File: skins/spring-dao.scss (hunk 1 above)
Comment (AlexN-OceanDAO) [reply skins/spring-dao.scss]: Yes. Using default. Made updates

File: spaces/spring-dao/index.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "SpringDAO",
+  "network": "1",
+  "symbol": "SPRNG",
+
```
Comment (bonustrack) [inline spaces/spring-dao/index.json:5]: Can you remove this line
```suggestion
```

Comment (bonustrack) [reply spaces/spring-dao/index.json:5]: Then your PR should be GTG

**Review by AlexN-OceanDAO — COMMENTED** [review]
line 5 deleted

File: spaces/spring-dao/index.json (hunk 2 above)
Comment (AlexN-OceanDAO) [reply spaces/spring-dao/index.json]: Great. Deleted. Thank you!

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**